config/sync/dblog.settings.yml

```diff
@@ -0,0 +1,3 @@
+row_limit: 1000
```

Comment:
I'd suggest a slightly higher amount... 10k? CSP reports would fill this rather quickly in the short term. 10k rows shouldn't take up much disk space on the db service.

Reply:
Yes, another good spot !